app/components/candidate_interface/course_choices_review_component.rb

```diff
@@ -44,42 +44,46 @@ def show_missing_banner?
       @show_incomplete && !@application_form.course_choices_completed && @editable
     end
 
+    def course_change_path(course_choice)
+      if FeatureFlag.active?('edit_course_choices') && has_multiple_courses?(course_choice)
+        candidate_interface_course_choices_course_path(
+          course_choice.provider.id,
+          course_choice_id: course_choice.id,
+        )
+      end
+    end
+
+    def site_change_path(course_choice)
+      if FeatureFlag.active?('edit_course_choices') && has_multiple_sites?(course_choice)
+        candidate_interface_course_choices_site_path(
+          course_choice.provider.id,
+          course_choice.course.id,
+          course_choice.offered_option.study_mode,
+          course_choice_id: course_choice.id,
+        )
+      end
+    end
+
   private
 
     attr_reader :application_form
 
     def course_row(course_choice)
       url = "https://www.find-postgraduate-teacher-training.service.gov.uk/course/#{course_choice.provider.code}/#{course_choice.course.code}"
```

Comment:
🆕 🆕 🆕 https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1834/files#diff-72aea198f50b1d4f2407a8f9f4c904f6R59

```suggestion
      url = course.find_url
```

(feel free to remove now redundant variable...)